frontend/VolunteerOne/components/EventCard/CardBody.js

```diff
@@ -0,0 +1,55 @@
+import { StyleSheet } from "react-native";
+import { Block, Text } from "galio-framework";
+
+/*
+Description:
+  This component returns the body of a card. It receives the data to insert as a dictionary. 
+  When receiving the data, it must have a "type" key attribute to be able to 
+  dynamically generate the content of the body. 
+  Example types:
+    "event": organization posting an event 
+    "announcment": organization posting an announcment 
+Props received:
+  data - dictionary of information to insert into card
+*/
+const CardBody = ({ data }) => {
+  //retreiving type
+  const type = data["type"];
+  //body for an event posting
+  if (type == "event") {
+    return (
+      <Block>
+        <Text style={styles.eventName}>{data["name"]}</Text>
+        <Text>
+          {data["subject"]}
+          {"\n\n"}
+          {data["description"]}
+          {"\n\n"}
+          When: {data["date"]} {"\n\n"}
```

Comment:
In the test app, When/Where currently displays even when there's no data. Is it possible to only display these two lines when data["date"] / data["location"] are not empty strings?
```suggestion
          {data["date"] != "" ? <Text> When: {data["date"]} {"\n\n"} </Text> : <Text>{"\n\n"}</Text> }
```

Reply:
Yeah we could do that. I just have it hard-coded like that for now so we can see what it looks like. I can add that change in the future